### ISSUE TITLE
acceptance rate not calculated by compute_mcmc

### ISSUE THREAD

**Issue by flipoyo** [flipoyo/MOLONARI1D#156]
the acceptance rate is always egal to 1.  when displayed.

**Comment by MartinJessenne** [comment]
I fixed a problem in the DREAM MCMC computation, it appeared that the temperatures profiles were always the same, hence the constant approval rate of 1. This was due to the fact that my method of using deepcopies to handle multichain calculation was conflicting with the direct model whose decorator Checker is poorly handling the usage of deep copies. I provided a quick fix whenever the direct model is called, but maybe it will require another issue to solve the problem at the level of the Checker decorator. This helped to solve partially the issue with the quantiles computations. Now it appears that the direct model is seldom providing temperatures profiles with NaN values. I also provided a quick fix making sure that an iteration is accepted if there isn't any NaN value in the profiles but once again I think it is worth opening a new issue to check if the direct model is correctly working (another hint would be the weird shape of the umbrella plot for multilayer cases).
